browsing-context.md

```diff
@@ -0,0 +1,28 @@
+# Prerendering browsing contexts
+
+TODO placeholder. Grab stuff from portals.
+
+* apply mitigations as above to subresource and scripted fetches
+* deny scripted access to unpartitioned storage, such as cookies and IndexedDB
+* deny permission to invoke `window.alert`, autoplay audio, and other APIs inappropriate at this time
+
+JS API probably belongs here (maybe it should use page visibility API instead):
```

Comment:
Yeah, possibly. I think the utility of a dedicated API basically depends on how many different kinds of modes there are that authors need to distinguish.